savant/deepstream/utils/object.py

```diff
@@ -173,3 +174,11 @@ def nvds_get_obj_bbox(nvds_obj_meta: pyds.NvDsFrameMeta) -> Union[NvDsBBox, NvDs
         return NvDsRBBox(nvds_obj_meta)
 
     raise IncorrectSelectionType('Unsupported object selection type.')
+
+
+def nvds_get_obj_custom_data_struct(obj_meta):
```

Comment:
Rename to nvds_get_obj_draw_label and use custom meta_type (see my comment below)

Reply:
Refactored, added custom user meta type.